BrowserKit/Sources/Common/Constants/StandardImageIdentifiers.swift

```diff
@@ -34,23 +34,19 @@ public struct StandardImageIdentifiers {
         public static let privateModeCircleFillPurple = "privateModeCircleFillMediumPurple"
         public static let readerView = "readerViewMedium"
         public static let share = "shareMedium"
-        public static let starFill = "starFillMedium"
```

Comment:
I am not sure we can clean those up standard image identifiers? cc @thatswinnie ?